src/components/OrderableList/useDraggableItem.tsx

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { DraggableItemProps, DraggableItemResult, useDrag } from "@react-aria/dnd";
+import { DraggableCollectionState } from "@react-stately/dnd";
+
+export const useDraggableItem = (props: DraggableItemProps, state: DraggableCollectionState): DraggableItemResult => {
+    const { dragProps, dragButtonProps } = useDrag({
+        getAllowedDropOperations() {
+            return ["move"];
+        },
+        getItems() {
+            return state.getItems(props.key);
+        },
+        renderPreview() {
+            return state.renderPreview(props.key);
+        },
+        onDragStart(e) {
+            state.startDrag(props.key, e);
+        },
+        onDragMove(e) {
+            state.moveDrag(e);
+        },
+        onDragEnd(e) {
+            state.endDrag(e);
+        },
+    });
+
+    return {
+        dragProps,
+        dragButtonProps: {
+            ...dragButtonProps,
```

Comment:
Seems like you are missing the `aria-label`
https://github.com/adobe/react-spectrum/blob/bc86137330c5cb71ae3866f78f95e96ce74d4d61/packages/%40react-aria/dnd/src/useDraggableItem.ts#L64

Reply:
fixed